docs/networking.md

```diff
@@ -290,6 +291,38 @@ Romana uses the cluster's etcd as a backend for storing information about routes
 This does not affect normal etcd operations or require special treatment when upgrading etcd.
 The etcd port (4001) is opened between masters and nodes when using this networking option.
 
+#### Amazon VPC Backend
+
+The [Amzon VPC CNI](https://github.com/aws/amazon-vpc-cni-k8s) plugin
+requires no additional configurations to be done by user. 
+
+**Important:** the pods uses the VPC CIDR, i.e. there is no isolation between the master, node/s and the internal k8s network.
+
+**Note:** the next are the required permissions to run the plugin:
```

Comment:
> the next are the required permissions to run the plugin:

How about

The following permissions are added to all nodes by kops to run the provider.

Reply:
done